### PR TITLE
Added brief documentation on setting up API server to run as a WSGI app.

### PR DESCRIPTION
**Pull request by dspruell**
Please review. See #517.

### CODE REVIEW

**Comment by jbremer** [comment]
Looking good @dspruell, thanks :)
@jekil are we good to merge this?

File: docs/book/src/usage/api.rst [hunk 1]
```diff
@@ -27,6 +27,96 @@ By default it will bind the service on **localhost:8090**. If you want to change
...
+
+   ``/run/uwsgi/app/cuckoo-api/socket``
+
+Nginx setup
+^^^^^^^^^^^
+
+With the API server running in uWSGI, Nginx can now be set up to run as a web
+server/reverse proxy, backending HTTP requests to it.
+
+To begin, create a Nginx configuration file at ``/etc/uwsgi/apps-available/cuckoo-api.ini``::
```
Comment (jbremer) [inline docs/book/src/usage/api.rst]: This should be ``/etc/nginx/sites-available/cuckoo-api`` as used later as well :)

**Comment by dspruell** [comment]
Good catch. :-|  Path updated.

**Comment by botherder** [comment]
Looks good, let's merge.

**Pull request merged**